### PR TITLE
Add build directory to container

### PR DESCRIPTION
**Pull request by Elkrival**
The production container needs an app_build directory and it the scripts need the app_build directory for the build.

### CODE REVIEW

File: singularity/Singularity [hunk 1]
```diff
@@ -60,7 +60,7 @@ gpgkey=https://www.mongodb.org/static/pgp/server-4.4.asc" >> /etc/yum.repos.d/mo
     ## clone the DPdash digital phenotyping dashboard
     git clone https://github.com/PREDICT-DPACC/dpdash.git
     cd dpdash
-
+    mkdir app_build
```
Comment (tashrifbillah) [inline singularity/Singularity:63]: Okay so why don't we `mkdir public` instead of having to change all following?

Comment (Elkrival) [reply singularity/Singularity:63]: The project has a public directory where the index.html file lives, this directory is where the javascript lives but it gets created when the application is built by webpack, it seems that the error we received is webpack missing the permissions necessary to create a directory on the container.

**Review by mikestone14 — APPROVED** [review]

File: singularity/Singularity (hunk 1 above)
Comment (tashrifbillah) [reply singularity/Singularity:63]: Does not make sense to me. We have been building DPdash forever with this `public/` directory in place. We should not have to rename that to `app_build/` today.

File: singularity/init.sh [hunk 2]
```diff
@@ -139,7 +139,7 @@ echo '***************Setting up DPdash*****************'
 singularity exec \
 -B ${containerDataDir}:/data \
 -B ${containerDataDir}/dpdash/configs/dashboard:/sw/apps/dpdash/server/configs \
--B ${containerDataDir}/dpdash/webpack-build:/sw/apps/dpdash/public/js \
+-B ${containerDataDir}/dpdash/webpack-build:/sw/apps/dpdash/app_build/js\
```
Comment (tashrifbillah) [inline singularity/init.sh:142]: Space missing after `js`.

Comment (tashrifbillah) [reply singularity/init.sh:142]: Even after I provide space, we have more errors:
```
WARNING: skipping mount of /dpdash/webpack-build: stat /dpdash/webpack-build: no such file or directory
FATAL:   container creation failed: mount hook function failure: mount /dpdash/configs/dashboard->/sw/apps/dpdash/server/configs error: while mounting /dpdash/configs/dashboard: mount source /dpdash/configs/dashboard doesn't exist

```

**Pull request merged**